src/obfl-specification.html

```diff
@@ -1225,6 +1225,29 @@ <h4 id="L1023" class="include">block</h4>
             left at the top of a page.</dd>
           <dd>An integer in the range [1, 9].</dd>
           <dd>Default is 1.</dd>
+        <dt>display-when [optional]</dt>
+          <dd>An condition that determines whether the contents of the block are
+            rendered.</dd>
+          <dd>Default is true.</dd>
+          <dd>All markers and in this block and its descendant elements will be
+            processed, regardless of the outcome of the condition. So it is
+            possible to use a marker's value even when the block is not
+            rendered.</dd>
+          <dd>If this block (or a descendant element) contains an anchor and the
+            block is not displayed, then the corresponding item is not
+            displayed as well.</dd>
```

Comment:
... then the corresponding item is not displayed ~as well~ either.

Reply:
Solved.